lib/functions/inline/arise_logo.sh

```diff
@@ -9,7 +9,7 @@
 # arise_logo
 
 arise_logo() {
-cat <<EOF
+	cat <<"EOF"
```

Comment:
no reason to enable variable expansion here
```suggestion
	cat <<EOF
```

Reply:
putting it in quotes is what disables variable expansion